spec/unit/actions/synchronize_ldm_spec.rb

```diff
@@ -0,0 +1,57 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+describe GoodData::LCM2::SynchronizeLdm do
+  let(:gdc_gd_client) { double(GoodData::Rest::Client) }
+  let(:logger) { double(Logger) }
+  let(:project) { double(GoodData::Project) }
+  let(:synchronize) do
+    [{ from: 'foo', to: [] }]
+  end
+  let(:basic_params) do
+    {
+        gdc_gd_client: gdc_gd_client,
+        development_client: gdc_gd_client,
+        synchronize: synchronize,
+        gdc_logger: logger
+    }
+  end
+  let(:params) { basic_params }
+  let(:converted_params) do
+    GoodData::LCM2.convert_to_smart_hash(params)
+  end
+
+  before do
+    allow(gdc_gd_client).to receive(:projects).and_return(project)
+    allow(project).to receive(:title)
+    allow(project).to receive(:pid)
+    allow(logger).to receive(:info)
+  end
+
+  context 'when include_computed_attributes is not specified' do
+    let(:params) { basic_params.merge(include_computed_attributes: 'true') }
+    it 'diffs LDM with computed attributes' do
+      expect(project).to receive(:blueprint).with(include_ca: true)
+      subject.class.call(converted_params)
+    end
+  end
+
+  context 'when include_computed_attributes is true' do
```

Comment:
Similar code found in synchronize_ldm_spec.rb:34, synchronize_ldm_spec.rb:42 (mass = 58)